Project3/index.html

```diff
@@ -0,0 +1,211 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles.css">
+
+    <link href="https://fonts.googleapis.com/css?family=Noto+Sans" rel="stylesheet">
+    <title>GD2 Project 3</title>
+</head>
+
+<body>
+    <header>
+        <h1>My Day: March 25th</h1>
+    </header>
+
+
+    <div class="container">
```

Comment:
Could swap this `div` for a `main`?

```suggestion
    <main class="container">
```